test/conformance/conformance_diff_test.go

```diff
@@ -0,0 +1,131 @@
+package conformance
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/bom-squad/protobom/pkg/formats"
+	"github.com/bom-squad/protobom/pkg/reader"
+	"github.com/bom-squad/protobom/pkg/sbom"
+	"github.com/bom-squad/protobom/pkg/writer"
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/require"
+)
+
+type fakeBuf struct {
+	*bufio.ReadWriter
+}
+
+func (fwc *fakeBuf) Close() error {
+	return nil
+}
+
+func (f *fakeBuf) Seek(offset int64, whence int) (int64, error) {
+	return 0, nil
+}
+
+func TestDiffTranslationFormats(t *testing.T) {
+	from := formats.SPDX23JSON
+	to := formats.CDX14JSON
+
```

Comment:
Remember that we don't care about going from one format to another. protobom restricts its opinions to/from the native formats to the neutral representation.